backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/cgen/CBridgeGen.kt

```diff
@@ -306,7 +310,11 @@ internal fun KotlinStubs.generateObjCCall(
         receiver: ObjCCallReceiver,
         arguments: List<IrExpression?>
 ) = builder.irBlock {
-    val callBuilder = KotlinToCCallBuilder(builder, this@generateObjCCall, isObjCMethod = true)
+    val exceptionMode = ForeignExceptionMode.byValue(
```

Comment:
Does this handle the case when manifest doesn't contain the property? This is required to make the compiler able to consume older "cinterop klibs".

Reply:
(Should be ok, I probably misread the `byValue` implementation).

Reply:
Yes it does. `ForeignExceptionMode.byValue` returns default value in that case (i.e. "terminate") . BTW this case occasionally tested a lot because all standard frameworks compiled without that option so they handled as before.